integrations/http/server.go

```diff
@@ -23,16 +29,21 @@ type HTTPHandler struct {
 
 func Start(l *zap.Logger, mux *http.ServeMux, d sdkservices.Dispatcher) {
 	h := HTTPHandler{dispatcher: d, logger: l}
-	mux.Handle(routePrefix, http.StripPrefix(routePrefix, h))
+	mux.Handle(routePrefix("{ns}")+"*", h)
 }
 
 func (h HTTPHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	l := h.logger.With(zap.String("url", r.URL.String()))
 
-	l.Info("Incoming request")
+	l.Info("incoming request")
```

Comment:
We need to standardize on logging conventions throughout the code - levels, text, and args - but not a concern for this PR.